crates/win-api-wrappers/src/wts.rs

```diff
@@ -20,78 +19,77 @@ use crate::ui::MessageBoxResult;
 use crate::utils::{AnsiString, SafeWindowsString};
 
 /// RAII wrapper for WTS virtual channel handle.
-pub struct WTSVirtualChannel(HANDLE);
+pub struct WtsVirtualChannel(HANDLE);
```

Comment:
Rust naming convention:

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. 

https://rust-lang.github.io/api-guidelines/naming.html